app/controllers/reminders.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  actions: {
+    removeReminder(reminder) {
+      reminder.deleteRecord();
+      reminder.save();
```

Comment:
destroyRecord deletes and saves therefore no need to delete and then save on line below. Thanks for the tip. @Tman22